flows/waits/time.go

```diff
@@ -0,0 +1,25 @@
+package waits
+
+import (
+	"github.com/nyaruka/goflow/flows"
+	"github.com/nyaruka/goflow/flows/events"
+)
+
+const TypeTime string = "time"
+
+type TimeWait struct {
+	Timeout int `json:"timeout" validation:"required,gte=0"`
+}
+
+func (w *TimeWait) Type() string { return TypeMsg }
+
+func (w *TimeWait) Apply(run flows.FlowRun, step flows.Step) {
+	run.ApplyEvent(step, nil, &events.TimeWaitEvent{Timeout: w.Timeout})
+}
+
+// CanResume returns true for a message wait if a message has now been received on this step
+func (w *TimeWait) CanResume(run flows.FlowRun, step flows.Step) bool {
+	return true
```

Comment:
Ok well I originally had this as a null wait but then figured if I added a `Timeout` field then we could solve two problems with one wait (timeout=0 for a null wait). I'll change it back to `NullWait` and we can solve the timeout issue for time and message waits later.